_rules/video-captions-correct-a3b9xz.md

```diff
@@ -0,0 +1,160 @@
+---
+id: a3b9xz
+name: Video element auditory content has correct captions
+rules_format: 1.1
+rule_type: atomic
+description: |
+  This rule checks that provided captions are correct for audio information in non-streaming `video` elements.
+accessibility_requirements:
+  wcag20:1.2.2: # Captions (Prerecorded) (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.2.4: # Captions (Live) (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
```

Comment:
The applicability ignores non-streaming videos. I think that's the right call here, the same video cannot fail both criteria, so you'll need to remove 1.2.4 from this.

```suggestion
```